Sources/GRPCHealthService/HealthService+Service.swift

```diff
@@ -42,6 +42,24 @@ extension HealthService.Service {
     return ServerResponse(message: response)
   }
 
+  func list(
+    request: ServerRequest<Grpc_Health_V1_HealthListRequest>,
+    context: ServerContext
+  ) async throws -> ServerResponse<Grpc_Health_V1_HealthListResponse> {
+    let serviceStatuses = self.state.listStatuses()
+
+    var listResponse = Grpc_Health_V1_HealthListResponse()
+
+    for (service, status) in serviceStatuses {
+      var checkResponse = Grpc_Health_V1_HealthCheckResponse()
+      checkResponse.status = status
+
+      listResponse.statuses[service] = checkResponse
```

Comment:
FYI protobuf messages have a `static` `with` method allowing you to populate them in a closure. This composes nicely with type-inference allowing you to just write:

```suggestion
      listResponse.statuses[service] = .with { response in
        response.status = status
      }
```